### ISSUE TITLE
'High Contrast Black' theme Inline Help WCAG AAA contrast compliance

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1007]
**Describe the bug**
The  Inline Help components' trigger button color contrast ratio is not 7:1 or above.

> What version of the Fundamental Library Styles are you using?
v0.9.1-rc.0

> What front-end framework are you implementing in, e.g., Angular, React, etc.?
React

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/inline-help.html
2. Check the trigger button color contrast

**Expected behavior**
If we want WCAG AAA compliance, this contrast ratio should be 7:1 or above.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81747523-1ac1cf80-945d-11ea-9c2a-130af685ea04.png)
![image](https://user-images.githubusercontent.com/43764832/81747592-34fbad80-945d-11ea-8736-e402edc5f967.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
This component is non-Fiori and has been deprecated and no longer exists.
